docs/pages/desktop-access/getting-started.mdx

```diff
@@ -516,12 +516,14 @@ spec:
   allow:
     windows_desktop_labels:
       "*": "*"
-    windows_desktop_logins: ["Administrator"]
+    windows_desktop_logins: ["jsmith"]
 ```
 
 <Admonition type="warning" title="RBAC Configuration">
 Ensure that each Teleport user is only assigned Windows logins that they should
 be allowed to access.
+
+Usernames shared between domain and local users may create login conflicts.
```

Comment:
Is there anything we can say about when shared usernames will create login conflicts? Or is the idea that it's unpredictable/arbitrary and operators should avoid it as a result?

Reply:
I'm not sure what you mean. The 'when' is when the username is the same between local and domain names.

Reply:
Just to clarify my original comment, I think the use of "may" is potentially confusing—are there situations where shared usernames don't create login conflicts? If not, we should replace "may" with "will".